test/extensions/bootstrap/wasm/test_data/logging_rust.rs

```diff
@@ -6,44 +6,45 @@ use proxy_wasm;
 #[no_mangle]
 pub fn proxy_abi_version_0_1_0() {}
 
-use log::{debug, error, info, trace, warn};
+use log::*;
```

Comment:
Maybe, `log` crate has private helper macros. In Rust 2018, we can available external macros with `use` syntax. But it works only there is no private helper macros. We should import them when we use log macros. But it will cause more complexity and confusion. In this case, we don't have any problem to use `*`. https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html#local-helper-macros